src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  height: 16px;
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) no-repeat 50%;
+  }
+
+  &--4 :nth-last-child(n + 2) {
```

Comment:
not just --4 should work (

If change stars--4 to stars--3, should be 3 active stars